src/pages/galleries/index.vue

```diff
@@ -11,15 +11,33 @@
           deck
           data-qa="gallery foyer"
         >
-          <ContentCard
-            v-for="gallery in galleries"
-            :key="gallery.identifier"
-            :title="gallery.name"
-            :url="{ name: 'galleries-all', params: { pathMatch: gallery.identifier } }"
-            :image-url="gallery.hasPartCollection.items[0] && imageUrl(gallery.hasPartCollection.items[0])"
-            :texts="[gallery.description]"
-            :show-subtitle="false"
-          />
+          <template
+            v-if="setGalleriesEnabled"
```

Comment:
Can the v-if and v-else be moved onto the ContentCard component element, making the template elements redundant?

Reply:
it goes a bit crazy when you do it like that since the content cards are looped through with v-for.
You get  the else output 20 times. So this is a bit of a workaround for that.